src/Orleans/Configuration/Options/StatisticsOptions.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Orleans.Runtime;
+using Orleans.Runtime.Configuration;
+using System;
+
+namespace Orleans.Configuration
+{
+    public class StatisticsOptions
```

Comment:
I think there should be an extension method on ISiloHostBuilder to configure Statistics, for better user experience. 
Based on this PR, the two way to configure statictics. One is using `AddLegacyNodeConfigurationSupport`, which is the legacy way. Second is configure a `StatisticOptions `into DI directly, which is not very straight for user .

Reply:
This is just my opinion, and I don't feel that strongly about it, but I don't believe we should add an extension method for each typed Options. It would make it more discoverable, for sure, but still I don't know, as long as the Options types are in the `Orleans.Hosting` namespace, it might be enough.
We can always add it later for convenience.